### ISSUE TITLE
Add ability to voice control Home Assistant scripts

### ISSUE THREAD

**Issue by JarvyJ** [JarvyJ/HomeIntent#235]
I think this could be most easily done via Home Assistant settings. This might be one of the few cases where we don't want to auto-import all the scripts... 

I was initially thinking to handle it via customizations, but it would be a little odd. I would have to pull in all the scripts and setup dummy sentences. Possible alter the code to default_disable all of them. Then folks would have to enable on each script and setup sentence aliases, but couldn't customize responses...

Since we can't really come up with sentences nor responses automatically, so it's probably best to leave it to the user. So I think putting it in the Home Assistant settings might be the best option:

```yaml
home_assistant:
  scripts:
    script.play_sunday_tunes:
      sentences:
        - "Put on the morning tunes"
        - "It's sunday morning"
      response: "Turning on the sunday music"
```

Users would have full control and could setup whatever they wanted!

**Comment by JarvyJ** [comment]
Work has started in this in #239 and it can work via the backend, I'm currently writing up stories that would be nice to haves related to the frontend.

**Comment by JarvyJ** [comment]
I'm going to merge in #239 soon and we shall see how it goes in the frontend before closing this out. It might end up as another issue.

**Comment by JarvyJ** [comment]
This got merged in with #259 and testing around it looks fairly promising! Closing this out for now
